### DISCUSSION TITLE
Can't recognise my student email, even though the uni is recognised

### DISCUSSION THREAD

**Discussion by jacksonhmg** [community/community#51855] [GitHub Education]
### Select Topic Area

Bug

### Body

Applying for student benefits in GitHub Education:

![imageedit_3_9458358596](https://user-images.githubusercontent.com/103947019/229560651-ee0f70fd-b680-4f0a-89bf-4a9fdbb91a69.png)

The student email at my university is not properly recognised! The correct format for it is shown in the above screenshot

When I try to use it anyway I get this:

![Screenshot 2023-04-03 234622](https://user-images.githubusercontent.com/103947019/229560988-48d57b45-6528-4576-b340-7e85f0667dc6.png)

**Comment by ramazancetinkaya** [comment]
@jacksonhmg, please kindly raise a ticket to [GitHub Support](https://support.github.com/) for the support engineers to investigate the issue.

They usually respond pretty quickly (within a couple of working days).

Hope this clarifies.

If you find this reply useful, please kindly 'Mark as answer' so that others know this issue is already addressed.

Have a good day.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @jacksonhmg ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by samus-aran** [comment]
Hi @jacksonhmg 

We've updated the details of your selected school. You should now be able to use your current email address. Let us know how you got on 👍
